script/configs/allowed_unpinned_deps.yaml

```diff
@@ -32,9 +31,11 @@
 - build
 - build_config
 - build_runner
+- build_test
 - collection
 - convert
 - crypto
+- dart_style
```

Comment:
Are you using this to validate that formatting doesn't change? If so, is there a reason you aren't just calling `dart format`?

Also, are you sure that any change in format output is a breaking change in `dart_style`? Because if it's not, you would want to pin this if you go this route.

Reply:
The test take the dart code as Library object and send it through code generator which produced a plan String output. 
I then use dart_style

Reply:
Please also move this allowance to the pinned dependencies section then, since it's likely that anyone using it should be pinning to avoid OOB errors.

Reply:
done also update the comment on top, seems like a typo